Filters/Widget/Choice/SingleTermChoice.php

```diff
@@ -77,10 +77,8 @@ public function preProcessSearch(Search $search, Search $relatedSearch, FilterSt
         if ($this->getSortType()) {
             $aggregation->setOrder($this->getSortType()['type'], $this->getSortType()['order']);
         }
-        
-        if ($this->getSize()) {
-            $aggregation->setSize($this->getSize());
-        }
+
+        $aggregation->setSize($this->getSize() > 0 ? $this->getSize() : 0);
```

Comment:
I know it looks sexy, but actually it confuses when you look at the code quickly. Please change to longer manner.